### DISCUSSION TITLE
how to tranfer the scvelo arrow of velocity into seurat to visulize?

### DISCUSSION THREAD

**Discussion by honghh2018** [theislab/scvelo#479] [Q&A]
Hi @ALL,
    Thanks to develop the great tools for RNA velocity analysis.
     On my case, i hope to  tranfer the velocity arrow into the seurat to visulize the UMAP with the scvelo arrow using R language.
     Best,
     hanhuihong

**Comment by WeilerP** [comment]
@honghh2018, you can either
1. transfer the your final AnnData object to a Seurat object and follow the implementation of `scv.pl.velocity_embedding` [here](https://github.com/theislab/scvelo/blob/1659cc8e00a45fcf87cd80a7013aae5531744613/scvelo/plotting/velocity_embedding.py#L13-L260), or
2. Calculate the start and end point of the velocity vectors as in `scv.pl.velocity_embedding` [here](https://github.com/theislab/scvelo/blob/1659cc8e00a45fcf87cd80a7013aae5531744613/scvelo/plotting/velocity_embedding.py#L13-L260) in Python and then export the objects to use in R.
